### DISCUSSION TITLE
Close entire app

### DISCUSSION THREAD

**Discussion by beppo-ivel** [appcelerator/titanium_mobile#13033] [Q&A] [answered]
I want to close the index view or rather the entire app after dialog is canceled. But the result of `$.index.close();` u can see in the picture below.
![closeApp](https://user-images.githubusercontent.com/72635809/130367077-7f2960f8-b540-4ef0-8c3c-72155e67b6fa.png)

controllers/index.js
```
GeoLocationUtil.requestPermissions()
.then(function(success){
	Ti.API.error(JSON.stringify(success));
	$.index.open();

})
.catch(function(error){
	Ti.API.error(JSON.stringify(error));
	$.index.close();
});
```

**Comment by m1ga** [comment, marked as answer]
you have to set `exitOnClose: true` to the index window.
Also checkout this SO post: https://stackoverflow.com/questions/13605653/exit-from-application-via-button-in-titanium/13618254#13618254

And if I remember correctly it is not possible to quit an iOS app via code.
